WebRoot/js/zimbraAdmin/globalconfig/view/GlobalConfigXFormView.js

```diff
@@ -165,6 +165,33 @@ GlobalConfigXFormView.registeredDeviceSelectionListener = function() {
     this.getForm().setInstanceValue(this.getSelection(), ZaGlobalConfig.A2_registeredDevice_Selection);
 }
 
+GlobalConfigXFormView.shouldEnableSyncButton = function(button) {
+    var sel = this.getForm().getInstanceValue(ZaGlobalConfig.A2_registeredDevice_Selection);
+
+    if (!sel || !sel[0]) {
+        return;
+    }
+    let statusArray = [];
+
+    if(button === ZaRegisterDevice.RD_BT_REMOVE) {
```

Comment:
Why aren't we using switch block instead of such lengthy if..else block ?

Reply:
updated